assets/scss/components/_header.scss

```diff
@@ -1,33 +1,48 @@
-.header .btn {
-  display: none;
-}
-
 .header {
+  left: 0;
+  padding-block: var(--spacing-2);
   position: absolute;
   top: 0;
-  left: 0;
   width: 100%;
-  padding-block: 16px;
   z-index: 4;
-}
 
-.header.active {
-  position: fixed;
-  background-color: var(--blue);
-  animation: headerActive 0.5s ease forwards;
-}
+  &.active {
+    -webkit-animation: headerActive 0.5s ease forwards;
+    animation: headerActive 0.5s ease forwards;
+    background-color: var(--color-primary-600);
+    position: fixed;
+  }
 
-@keyframes headerActive {
-  0% {
-    transform: translateY(-100%);
+  @-webkit-keyframes headerActive {
```

Comment:
I think 2012 is over, we can ditch that prefix :D

Reply:
autoprefixer ;p but ofc i'll delete it